WebContent/js/components/JobInstance.js

```diff
@@ -95,18 +95,27 @@ class JobInstance extends React.Component {
         const statusStyleActive = { display: 'inline' };
         const statusStyleAbend = { color: 'red', display: 'inline' };
         const statusStyleComplete = { color: 'grey', display: 'inline' };
-        if (job.get('returnCode')) {
-            if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')) {
-                return (<div style={statusStyleAbend}> [{job.get('returnCode')}]</div>);
+        const errorReturnCode = ['abend', 'jcl error', 'sys fail', 'conv error', 'sec error'];
+        const completeReturnCode = ['cc', 'canceled'];
+        const jobStatus = job.get('status');
+        if (jobStatus) {
+            if (jobStatus.toLowerCase().includes('output')) {
+                const jobReturnCode = job.get('returnCode');
+                if (jobReturnCode) {
+                    const lowerCaseJobReturnCode = jobReturnCode.toLowerCase();
+                    if (errorReturnCode.find(v => { return lowerCaseJobReturnCode.includes(v); })) {
```

Comment:
One last change, I'm not happy with using v as the variable name passed to the arrow function, I think a more suitable name would be errorReturnCode. and for the arrow function for complete return codes it should be completeReturnCode.

This is just to stay consistent with the rest of the code base, Thanks.